ETC.tsv

```diff
@@ -10456,10 +10456,10 @@ ETC_20150428_011434	Recipe - Superior Rune Bracelet
 ETC_20150428_011435	Recipe - Star Bracelet
 ETC_20150428_011436	FAQ: Lodge{nl}1)You can change the name of your lodge by clicking "Lodge Settings". {nl}2)You can buy Lodge skins using NXP by clicking "Change Lodge".{nl}3)You select your desired channel from the Lodge.
 ETC_20150428_011437	FAQ: Character HP, SP, STA {nl} 1) You can check your character's information on the upper left side of the screen. {nl}2) STA: This is stamina. When your STA is 0, you can only walk and can't run. {nl} It is also used when learning certain attributes. {nl} You can recover STA by destroying Tree Root Crystals or by using Stamina Pills. {nl} Stamina Pills can be bought through shop or can be earned by completing quests or defeating boss monsters.
-ETC_20150428_011438	FAQ: Character Level and Class Level {nl}You can check the info at the lower part of the screen. {nl}2) Character Level: this is your character's level. {nl}3) Class Level: this is the level of the class you chose. {nl} You can advace to another class when you reach level 15.
-ETC_20150428_011439	FAQ: Quick Slot{nl}1) You can add items or skills to the quick slot on the lower part of your skill. {nl}2) Skills are automatically added to quick slot when you learn quests. {nl}3)You can use up to 40 quick slots. {nl}4) Quick slots can be locked or unlocked. {nl}You cannot add or remove items or skill in quick slot when it is locked.
+ETC_20150428_011438	FAQ: Character Level and Class Level {nl}You can check the info at the lower part of the screen. {nl}2) Character Level: this is your character's level. {nl}3) Class Level: this is the level of the class you chose. {nl} You can advance to another class when you reach level 15.
+ETC_20150428_011439	FAQ: Quick Slot{nl}1) You can add items or skills to the quick slot on the lower part of your skill. {nl}2) Skills are automatically added to quick slot when you learn skills. {nl}3)You can use up to 40 quick slots. {nl}4) Quick slots can be locked or unlocked. {nl}You cannot add or remove items or skill in quick slot when it is locked.
```

Comment:
Change "on the lower part of your skill" to " at the lower part of your screen."